agent/internal/core.go

```diff
@@ -29,6 +30,9 @@ func Run(parent context.Context, version string, opts options.Options) error {
 	wg := errgroupx.WithContext(ctx)
 
 	log.Trace("starting main agent process")
+
+	logger.SetLogrus(opts.Log)
```

Comment:
What happens if a log level isn't specified in the devcluster? Do we default to a specific level?

Reply:
Yea, default log level is specified [here](https://github.com/determined-ai/determined/blob/main/agent/internal/options/options.go#L28)